sklearn/preprocessing/discretization.py

```diff
@@ -52,19 +55,26 @@ class KBinsDiscretizer(BaseEstimator, TransformerMixin):
     dtype : number type, default=np.float
         Desired dtype of output.
 
+    strategy : {'uniform', 'quantile', 'kmeans'}, (default='quantile')
+        Strategy used to define the widths of the bins.
+
+        uniform
+            All bins in each feature have identical widths.
+        quantile
+            Widths are defined by a quantile transform, to have a uniform
+            number of samples in each bin.
+        kmeans
+            Widths are defined by a k-means on each features.
```

Comment:
features -> feature

Perhaps:
"Values in each bin have the same nearest center of a k-means cluster."